packages/node/src/integrations/http/index.ts

```diff
@@ -97,6 +98,10 @@ interface HttpOptions {
   };
 }
 
+const instrumentSentryHttpBeforeOtel = generateInstrumentOnce(`${INTEGRATION_NAME}.sentry-before-otel`, () => {
```

Comment:
Just saw this now, could you add the reasoning for this (that is in the PR description) as a comment here too? Was super confused why this exists, and it's not really clear from the code alone why this is necessary! @lforst